### DISCUSSION TITLE
Should we have t shirt sizes for font size?

### DISCUSSION THREAD

**Discussion by technikhil314** [argyleink/open-props#116] [General]
`--font-size-00` has value of `0.5rem` which does not really go hand in hand with the prop name. IMO. 
Hence Should we have t shirt sizes for font size?

Feel free to enlighten me if I am missing any context.

Something like

```
  --font-size-2xs: .5rem;
  --font-size-xs: .75rem;
  --font-size-s: 1rem;
  --font-size-m: 1.1rem;
  --font-size-l: 1.25rem;
  --font-size-xl: 1.5rem;
  --font-size-2xl: 2rem;
  --font-size-3xl: 2.5rem;
  --font-size-4xl: 3rem;
  --font-size-5xl: 3.5rem;
```

**Comment by argyleink** [comment]
i never really felt that t-shirt sizes scaled as well as raw numbers. the rest of OP uses a raw numbering system, dont think we should make exceptions for font size.

the reason font size 0 and 00 being in the "unsafe" category is because font sizes under 1 rem are headed right into "illegible" territory. 1rem represents the base font size your users want to read at, go under that value with caution. that reasonable reasoning? 🙂

**Reply by technikhil314** [reply]
Yeah that makes sense. 
I am just thinking if we can rename `00` to some more relevant name in that case. Even I am not able to come up with good naming. May be anything starting with 0 is less that 1. so 075 -> .75rem and 05  ->  .5rem sounds more relevant?
